src/lib/components/Form/Form.js

```diff
@@ -168,7 +168,7 @@ const Form = ({
 
   const errorOnLoad = function (title, error) {
     snackbar.showError(title, error);
-    navigate(navigateBack);
+    navigate(navigateBack.includes("window.history") ? window.history.back(): navigateBack);
```

Comment:
It should be define in a variable  and then use  & check the alternative solution